### DISCUSSION TITLE
Moving tufup development to an organization account

### DISCUSSION THREAD

**Discussion by JessicaTegner** [dennisvang/tufup#162] [Q&A]
hi @dennisvang 

Since we now have this main repo, and the tufup-example repo, would it make sense to move all tufup related stuff to an organization?
Thinking down the line, if we also will have documentation, plugins (like uploaders), or other things.

Thinking like this, to try to future proof this, in the case that you move away from python, or something else happens.

What's your thoughts on this.

**Comment by dennisvang** [comment]
>[...] would it make sense to move all tufup related stuff to an organization?

Hi @JessicaTegner, that's an excellent idea! :)

What do you think, should we just call the organization `tufup` as well?

**Comment by JessicaTegner** [comment]
Hi @dennisvang 
yes I do think that "tufup" would be a fitting name for it, since that's the main project name. Then from there we can have
* tufup/tufup - Main repo
* tufup/example - example repo.
* tufup/*-plugin - for any eventual plugins.

And so on.

**Reply by dennisvang** [reply]
![Screenshot 2024-10-21 at 21-10-51 Set up your organization](https://github.com/user-attachments/assets/55bf93e3-f974-4462-89bb-8c5e5ca74a73)


Unfortunately it turns out the name [tufup][1] is already taken.

Thinking about an alternative name, I started wondering if it might be better to move the project under the umbrella of an *existing* organization, such as [jazzband][2], or perhaps [theupdateframework][3], if they would be willing.

[1]: https://github.com/tufup
[2]: https://github.com/jazzband
[3]: https://github.com/theupdateframework
